ergo-lib/src/wallet/signing.rs

```diff
@@ -131,6 +131,7 @@ pub fn make_context(
         inputs: inputs_ir,
         pre_header: state_ctx.pre_header.clone(),
         extension: ContextExtension::empty(),
+        headers: Vec::new()
```

Comment:
@greenhat Seems that need to add headers field to https://github.com/ergoplatform/sigma-rust/blob/6aa986d04b8955bd6c284ecf09954abb2c03f2a4/ergo-lib/src/chain/ergo_state_context.rs#L6, right?

Reply:
Yes, that's right.